### DISCUSSION TITLE
Changing console output

### DISCUSSION THREAD

**Discussion by sqcred** [Minestom/Minestom#646] [Help] [answered]
Is it possible to remove the [main] and the (ServerProcessImpl.start) ?

[main] [19:12:39] (ServerProcessImpl.start) - INFO - Starting Minestom server.

**Comment by mworzala** [comment, marked as answer]
You can add your own log config. We use https://tinylog.org/v2/

**Reply by sqcred** [reply]
never min im dumb. thanks for the help
